packages/tools/devtools/devtools-view/api-report/devtools-view.api.md

```diff
@@ -18,6 +18,7 @@ export function DevtoolsPanel(props: DevtoolsPanelProps): React_2.ReactElement;
 // @public
 export interface DevtoolsPanelProps {
     messageRelay: IMessageRelay;
+    unloadCallback?: () => void;
```

Comment:
Do we need to include a logger?.flush() also here to clean up logger if devtool close?